spec/takeaway_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new }
+  let(:items)     { {Cheeseburger: 1, Fries: 2}}
+  let(:order)     {double (:order)}
+
+describe '#show_menu' do
+    it 'brings up list of dishes with prices' do
+      menu_items = "Cheeseburger £5.00, Fries £3.00, Milkshake £2.99"
+      expect(takeaway.show_menu).to eq(menu_items)
+    end
+    end
```

Comment:
Lint/BlockAlignment: `end` at 13, 4 is not aligned with `describe '#show_menu' do` at 8, 0.